frontend/src/query_builder/QueryVisualization.jsx

```diff
@@ -2,12 +2,12 @@ import React, { Component, PropTypes } from "react";
 
 import Icon from "metabase/components/Icon.jsx";
 import LoadingSpinner from 'metabase/components/LoadingSpinner.jsx';
-import QueryVisualizationTable from './QueryVisualizationTable.jsx';
-import QueryVisualizationChart from './QueryVisualizationChart.jsx';
 import QueryVisualizationObjectDetailTable from './QueryVisualizationObjectDetailTable.jsx';
 import RunButton from './RunButton.jsx';
 import VisualizationSettings from './VisualizationSettings.jsx';
 
+import Visualization from "../visualizations/Visualization.jsx";
```

Comment:
same thing with relative vs absolute paths here.  since that component isn't part of the query_builder i think it's nicer to use an absolute path and make that clear, so using `"metabase/visualizations/Visualization.jsx"` makes the code much easier to read in cases like this.